src/cockpit/channels/filesystem.py

```diff
@@ -393,6 +407,11 @@ def get_group(gid: int) -> 'str | int':
         }
         stat_getters = tuple((key, available_stat_getters.get(key, lambda _: None)) for key in attrs)
 
+        available_access_getters = {
+                'r-ok': lambda name, fd, follow: get_access(name, fd, os.R_OK, follow_symlinks=follow),
```

Comment:
How about having a map from `r-ok` to `os.R_OK` (and so on) instead, and removing one layer of indirection?

Reply:
Also: `x-ok`?